terratorch/tasks/wxc_downscaling_task.py

```diff
@@ -31,6 +31,8 @@ def __init__(
 
         self.model_factory = get_factory(model_factory)
         self.model_config = model_config
+        # TODO Unify it with self.hparams
+        self.extended_hparams = self.model_config.to_dict()
```

Comment:
not quite sure why this is needed. if we want to add these to hparams, can we do `self.hparams |= self.model_config.to_dict()`?

Reply:
We could, but `self.hparams` is a `lightning.fabric.utilities.data.AttributeDict instance`.